Package/Assets/GDK-Tools/Source/Editor/GdkPlugin.cs

```diff
@@ -13,13 +13,13 @@ public static class GdkPlugin
         [MenuItem("GDK/Documentation/Developer Portal (GDK)")]
         private static void OpenDeveloperPortal()
         {
-            Application.OpenURL("https://docs.microsoft.com/gaming/gdk/");
+            Application.OpenURL("https://aka.ms/gamedevdocs");
         }
 
         [MenuItem("GDK/Documentation/Unity Integration Documentation (GDK)")]
         private static void OpenUnityDocs()
         {
-            Application.OpenURL("https://docs.microsoft.com/gaming/gdk/_content/gc/get-started-with-pc-dev/get-started-with-unity-pc/gdk-unity-end-to-end-guide");
+            Application.OpenURL("http://aka.ms/gdk_unity");
```

Comment:
Would recommend updating to:

Application.OpenURL("https://aka.ms/gdk_unity");
```suggestion
            Application.OpenURL("https://aka.ms/gdk_unity");
```

Reply:
Good idea. I'll do that.